source/vsd/vsd_git.rst

```diff
@@ -53,4 +53,8 @@ Recommended branch names:
     * Considerable large changes such as the adoption of a ROS new version, large package changes, or a considerably new phisically tested vehicle ability, should change the first digit (as in version 1.0 to version 2.0), little changes such as untested in real life improvements change the second digit (such as in version 1.1 to version 1.2).
   
 
+Pull requests
+-------------
+
```

Comment:
Add a clear description of how to do pull requests

Reply:
yap